### PR TITLE
Update Genome uploader options

### PR DESCRIPTION
**Pull request by JamesJeffryes**
for GenBank and FastaGFF

### CODE REVIEW

File: ui/narrative/methods/import_genbank_as_genome_from_staging/spec.json [hunk 1]
```diff
@@ -128,12 +128,8 @@
           "target_property": "genetic_code"
         },
         {
-          "input_parameter": "generate_ids_if_needed",
```
Comment (Tianhao-Gu) [inline ui/narrative/methods/import_genbank_as_genome_from_staging/spec.json]: 'generate_ids_if_needed' in 'parameters' and display.yaml should also be removed.

File: ui/narrative/methods/import_genbank_as_genome_from_staging/spec.json [hunk 2]
```diff
@@ -128,12 +128,8 @@
...
-          "target_property": "generate_ids_if_needed"
-        },
-        {
-          "input_parameter": "exclude_ontologies",
-          "target_property": "exclude_ontologies"
+          "input_parameter": "generate_missing_genes",
```
Comment (Tianhao-Gu) [inline ui/narrative/methods/import_genbank_as_genome_from_staging/spec.json]: GFU will take 'generate_missing_genes' and ignore 'exclude_ontologies'? Was is just a parameter name change for GFU? I don't see GFU is taking generate_missing_genes, at least not for master branch.

**Comment by JamesJeffryes** [comment]
I should have left a more informative comment. The goal is to add a generate_missing_genes method for both gff & genbank. This parameter is not in the master branch of GFU yet but is in the "genome_refactor" branch. I'm also removing exclude_ontologies because the new GFU ignores this parameter (ontologies are much more compact now). We would not deploy this version in production until the GFU updates are finished (very close now) but I'd like to test them in CI

**Comment by Tianhao-Gu** [comment]
💯

**Pull request merged**